src/pages/management/configuration-api/v2.0/index.mdx

```diff
@@ -154,7 +154,13 @@ You can call the LiveChat API using librariers written in backend languages. To
   Did you write your own library and want it listed here? Let us know!
 </aside>
 
-# Agents
+# Agents (deprecated)
+
+<Warning>
+
+**Please note this API is deprecated**. We highly recommend switching to the new [Management APIs](/management/).
```

Comment:
Wouldn't this be a better place to link to? `/management/configuration-api#agents`

Reply:
You're right :)